src/app/components/ModalComponents/ModalDetailsTable.js

```diff
@@ -0,0 +1,94 @@
+import elementFactory from '../../utils/elementFactory';
+
+function chooseStyleAndAnswer(round) {
+  if (!round) {
+    return ['', ''];
+  }
+  return round.isCorrect
+    ? [round.answer, 'details__table__answer--correct']
+    : [round.answer, 'details__table__answer--incorrect'];
+}
+
+function ModalDetailsTable(quiz, humanData, computerData) {
+  const tableHeaders = ['', 'You', 'Computer', 'Answer'];
+  const newTable = elementFactory('table', {
+    className: 'details__table',
+  });
+  const tableHead = elementFactory('thead');
+  const tableHeadRow = elementFactory('tr');
+  tableHeaders.forEach((header) => {
+    const tableHeader = elementFactory(
+      'th',
+      {
+        className: 'details__table__head',
+      },
+      header,
+    );
+    tableHeadRow.appendChild(tableHeader);
+  });
+
+  // TABLE BODY
+  const tableBody = elementFactory('tbody', {
+    className: 'details__table__body',
+  });
+
+  const humanAnswers = humanData.detailedAnswers;
+  const computerAnswers = computerData.detailedAnswers;
+  const { questions } = quiz;
+  const rightAnswers = questions.reduce((listOfAnswers, question) => {
+    listOfAnswers.push(question.rightAnswer);
+    return listOfAnswers;
+  }, []);
```

Comment:
Will this work?
```suggestion
  const rightAnswers = questions.filter((question) => question.rightAnswer
```

Reply:
Filter didn't, but map did.

Reply:
that's right, but do you know why it didn't?

Reply:
Map returns something changed for each item, in this case it was the right answer. Filter checks if expression is true, but doesnt return a different outcome, and in this case it was the full question object.